xbmc/video/windows/GUIWindowVideoNav.h

```diff
@@ -49,7 +49,7 @@ class CGUIWindowVideoNav : public CGUIWindowVideoBase
   virtual bool OnAction(const CAction &action) override;
   virtual bool OnMessage(CGUIMessage& message) override;
 
-  virtual void OnItemInfo(const CFileItem& fileItem, ADDON::ScraperPtr &info) override;
+  virtual void OnItemInfo(CFileItem* fileItem, ADDON::ScraperPtr &info) override;
```

Comment:
watch using raw pointers to CFileItem's, most are really CFileItemPtr and are shared_ptr.  These have the tendency of slipping out from under you when you diddle with the raw pointer.